learning-javaqa/src/main/java/com/softvision/qa/automation/groupa/ExerciseThreeMod1.java

```diff
@@ -0,0 +1,57 @@
+package com.softvision.qa.automation.groupa;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+//Sa se citeasca un fisier din folderul de resources si sa se afiseze numarul de linii si ultima linie.
+
+public class ExerciseThreeMod1 {
+
+    public static void main(String[] args) throws IOException {
+
+        File file = new File("./resources/data.txt");
```

Comment:
asta nu e folderul de tip resources. el trebuie sa fie in src/..
Adauga un folder de tip resources. In cazul de fata e doar un folder in root (de aici si ./ inaintea folderului). Cand citim un fisier din folderul de resource folosim o methoda anume (hint read stream as resource).
In momentul in care se face jarul si se deploiaza pe diferite masini, citirea din fisier poate devenii problematica pt ca pathurile pot diferi. de accea fisierele si configs file etc se pun intrun fisier special. Daca nu ma insel la maven course era ceva despre resources

Reply:
Au fost facute modificarile si aici. Multumesc mult pentru ajutor!